### ISSUE TITLE
NAN

### ISSUE THREAD

**Issue by wangshuaicumt** [Unidata/netcdf4-python#1239]
ps_point = d1.sel(lon='117.17638888845157', lat='35.47979166666699')
print(ps_point.values)

When I run the codes, but the terminal shows NAN. How to solve it?

nan

**Comment by jswhit** [comment]
are you using xarray?

**Comment by wangshuaicumt** [comment]
> are you using xarray?

Yeah.

**Comment by wangshuaicumt** [comment]
> are you using xarray?

Is there a better way to achieve?

**Comment by jswhit** [comment]
this is really not a netcdf4-python issue.  I suspect you are select a point with missing or undefined data, but you would have to ask an xarray expert (maybe [here](https://groups.google.com/g/xarray)).

**Comment by wangshuaicumt** [comment]
> this is really not a netcdf4-python issue. I suspect you are select a point with missing or undefined data, but you would have to ask an xarray expert (maybe [here](https://groups.google.com/g/xarray)).

Yeah, thanks. The data I read out using xray is like this, and the graph drawn is as follows.

<xarray.DataArray 'z' (lat: 1180, lon: 1910)>
[2253800 values with dtype=float32]
Coordinates:
  * lon      (lon) float64 116.2 116.2 116.2 116.2 ... 117.2 117.2 117.2 117.2
  * lat      (lat) float64 35.48 35.48 35.48 35.48 ... 35.97 35.97 35.97 35.97
Attributes:
    long_name:     z
    actual_range:  [-153.15245056  136.86753845]

![Figure_1](https://user-images.githubusercontent.com/117328108/223599339-0066532c-cc14-4026-b60d-5dc3f77b5f69.png)

**Comment by mgrover1** [comment]
You should specify ds.sel(lat=.., lon=..., method='nearest' ), without that, it is trying to find that exact value.
